src/mappings/modules/system-stabilizer/flap.ts

```diff
@@ -64,4 +65,26 @@ export function handleTick(event: LogNote): void{
   auction.lastUpdate = event.block.timestamp
   
   auction.save()
+}
+
+export function handleYank(event: LogNote): void{
+  let id = bytes.toUnsignedInt(event.params.arg1)
+
+  let auction = Auctions.loadOrCreateAuction(id.toString()+"-0", event)
+  auction.active = false
+  auction.save()
+
+  let log = new EndedSurplusAuctionLog(event.transaction.hash.toHex() + '-' + event.logIndex.toString() + '-2')
```

Comment:
After seeing this code I realized that adding a new entity just for tracking an Auction deletion is too much. I guess it could be better to implement a soft delete approach. We can add an extra field to the Auction: `deletedAt: BigInt`